test/queues/schedules.int.spec.ts

```diff
@@ -0,0 +1,341 @@
+    // Cron function runs may persist after a test has finished
+    await wait(3000)
+    // Now we can destroy the payload instance
+    await payload.destroy()
+    _internal_resetJobSystemGlobals()
+  })
+
+  afterEach(() => {
+    _internal_resetJobSystemGlobals()
+  })
+
+  beforeEach(async () => {
+    // Set autorun to false during seed process to ensure no crons are scheduled, which may affect the tests
+    _internal_jobSystemGlobals.shouldAutoRun = false
+    _internal_jobSystemGlobals.shouldAutoSchedule = false
+    await clearAndSeedEverything(payload)
+    const data = await restClient
+      .POST('/users/login', {
+        body: JSON.stringify({
+          email,
+          password,
+        }),
+      })
+      .then((res) => res.json())
+
+    if (data.token) {
+      token = data.token
+    }
+    payload.config.jobs.deleteJobOnComplete = true
+    _internal_jobSystemGlobals.shouldAutoRun = true
+    _internal_jobSystemGlobals.shouldAutoSchedule = true
+  })
+
+  it('can auto-schedule through local API and autorun jobs', async () => {
+    // Do not call payload.jobs.queue() - the `EverySecond` task should be scheduled here
+    await payload.jobs.handleSchedules()
+
+    // Do not call payload.jobs.run{silent: true})
+
+    await waitUntilAutorunIsDone({
+      payload,
+      queue: 'autorunSecond',
+      onlyScheduled: true,
+    })
+
+    const allSimples = await payload.find({
+      collection: 'simple',
+      limit: 100,
+    })
+
+    expect(allSimples.totalDocs).toBe(1)
+    expect(allSimples?.docs?.[0]?.title).toBe('This task runs every second')
+  })
+
+  it('can auto-schedule through handleSchedules REST API and autorun jobs', async () => {
+    // Do not call payload.jobs.queue() - the `EverySecond` task should be scheduled here
+    await restClient.GET('/payload-jobs/handle-schedules', {
+      headers: {
+        Authorization: `JWT ${token}`,
+      },
+    })
+
+    // Do not call payload.jobs.run({silent: true})
+
+    await waitUntilAutorunIsDone({
+      payload,
+      queue: 'autorunSecond',
+      onlyScheduled: true,
+    })
+
+    const allSimples = await payload.find({
+      collection: 'simple',
+      limit: 100,
+    })
+
+    expect(allSimples.totalDocs).toBe(1)
+    expect(allSimples?.docs?.[0]?.title).toBe('This task runs every second')
+  })
+
+  it('can auto-schedule through run REST API and autorun jobs', async () => {
+    // Do not call payload.jobs.queue() - the `EverySecond` task should be scheduled here
+    await restClient.GET('/payload-jobs/run?silent=true', {
+      headers: {
+        Authorization: `JWT ${token}`,
+      },
+    })
+
+    await waitUntilAutorunIsDone({
+      payload,
+      queue: 'autorunSecond',
+      onlyScheduled: true,
+    })
+
+    const allSimples = await payload.find({
+      collection: 'simple',
+      limit: 100,
+    })
+
+    expect(allSimples.totalDocs).toBe(1) // TODO: FLAKE: Could be 2
```

Comment:
Is this test stable enough to merge?

Reply:
Yep - I removed the comment. This does not happen anymore